runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -753,3 +753,35 @@ J9::ARM64::CodeGenerator::inlineDirectCall(TR::Node *node, TR::Register *&result
    resultReg = NULL;
    return false;
    }
+
+TR::Instruction *J9::ARM64::TreeEvaluator::generateVFTMaskInstruction(TR::CodeGenerator *cg, TR::Node *node, TR::Register *dstReg, TR::Register *srcReg, TR::Instruction *preced)
+   {
+   TR_J9VMBase *fej9 = (TR_J9VMBase *)(cg->fe());
+   uintptrj_t mask = TR::Compiler->om.maskOfObjectVftField();
+#ifdef OMR_GC_COMPRESSED_POINTERS
```

Comment:
These compile-time `#ifdef`s are in the process of being deprecated in favour of compile-time checks (see the recent PRs from @gacholio).  I believe this code should simply be:
```
   bool isCompressed = TR::Compiler->om.compressObjectReferences();
```

Reply:
Updated the file reflecting the review comment.